pkg/utils/validation/features/featuregates.go

```diff
@@ -53,21 +53,21 @@ var featureGateVersionRanges = map[string]*FeatureGateVersionRange{
 	"CloudControllerManagerWebhook":                  {AddedInVersion: "1.27"},
 	"CloudDualStackNodeIPs":                          {AddedInVersion: "1.27"},
 	"ClusterTrustBundle":                             {AddedInVersion: "1.27"},
-	"CPUManager":                                     {LockedToDefaultInVersion: "1.26"},
+	"CPUManager":                                     {Default: true, LockedToDefaultInVersion: "1.26"},
 	"CPUManagerPolicyAlphaOptions":                   {AddedInVersion: "1.23"},
 	"CPUManagerPolicyBetaOptions":                    {AddedInVersion: "1.23"},
 	"CPUManagerPolicyOptions":                        {},
-	"CronJobTimeZone":                                {AddedInVersion: "1.24", LockedToDefaultInVersion: "1.27"},
+	"CronJobTimeZone":                                {Default: true, AddedInVersion: "1.24", LockedToDefaultInVersion: "1.27"},
 	"CSIInlineVolume":                                {Default: true, LockedToDefaultInVersion: "1.25", RemovedInVersion: "1.27"},
 	"CSIMigration":                                   {Default: true, LockedToDefaultInVersion: "1.25", RemovedInVersion: "1.27"},
 	"CSIMigrationAWS":                                {Default: true, LockedToDefaultInVersion: "1.25", RemovedInVersion: "1.27"},
 	"CSIMigrationAzureDisk":                          {Default: true, LockedToDefaultInVersion: "1.25", RemovedInVersion: "1.27"},
-	"CSIMigrationAzureFile":                          {LockedToDefaultInVersion: "1.27"},
+	"CSIMigrationAzureFile":                          {Default: true, LockedToDefaultInVersion: "1.26"},
```

Comment:
```suggestion
	"CSIMigrationAzureFile":                          {Default: true, LockedToDefaultInVersion: "1.27"},
```

Reply:
The report says `1.27` but according to the Kubernetes page it seems to be be `1.26`. That's weird 🤔 
```
CSIMigrationAzureFile | true | GA | 1.26 
```

Reply:
In the release notes of 1.27 it is mentioned
> The CSIMigrationAzureFile feature gate (for the feature which graduated to GA in v1.26) is now unconditionally enabled and will be removed in v1.28. (https://github.com/kubernetes/kubernetes/pull/114953, [@enj](https://github.com/enj))